util/src/main/java/org/folio/metastorage/util/OaiMetadataParserMarcInJson.java

```diff
@@ -0,0 +1,115 @@
+package org.folio.metastorage.util;
+
+import static org.folio.metastorage.util.Constants.CODE_LABEL;
+import static org.folio.metastorage.util.Constants.COLLECTION_LABEL;
+import static org.folio.metastorage.util.Constants.CONTROLFIELD_LABEL;
+import static org.folio.metastorage.util.Constants.LEADER_LABEL;
+import static org.folio.metastorage.util.Constants.SUBFIELD_LABEL;
+import static org.folio.metastorage.util.Constants.TAG_LABEL;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import javax.xml.stream.XMLStreamConstants;
+import javax.xml.stream.XMLStreamReader;
+
+
+public class OaiMetadataParserMarcInJson implements OaiMetadataParser<JsonObject> {
```

Comment:
Is there anything specific to OAI in this class or is it a general MarcXml parser?